mephisto/data_model/unit.py

```diff
@@ -181,7 +181,7 @@ def get_requester(self) -> "Requester":
 
     def clear_assigned_agent(self) -> None:
         """Clear the agent that is assigned to this unit"""
-        logger.debug(f"Clearing assigned agent {self.__agent} from {self}")
+        logger.debug(f"Clearing assigned agent {self.agent} from {self}")
```

Comment:
nit: `MTurkUnit` does not have `agent`

Reply:
Ack! I fixed this locally and didn't push - wanted this to be `self.agent_id` because we actually stopped using `self.__agent`